### PR TITLE
Create cosmoscan.yml

### PR DESCRIPTION
**Pull request by poomthiti**
initial setup for github actions and edit test case to adapt with the current version of Cosmoscan

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,4 +77,4 @@
       "cypress"
     ]
   }
-}
+}
```
Comment (evilpeach) [inline package.json:80]: add end line

**Review by evilpeach — CHANGES_REQUESTED** [review]
remove `.mp4` files and add to `.gitignore`

File: package.json (hunk 1 above)
Comment (poomthiti) [reply package.json]: done with prettier krub

**Comment by evilpeach** [comment]
Why does the test fail?

**Comment by poomthiti** [comment]
> @poomthiti Why does the test fail?

[Laozi Testnet, As of 4 Jan 2022, 12:15pm]
1. Undelegate with WeStaking failed. [Error: invalid version]
2. Redelegate from [WeStaking, Cosmostation] to other validators failed. [Error: an error occurred]

Everthing works with StakeWithUs, maybe will perform test cases on it instead

**Comment by poomthiti** [comment]
[Test Passed] Changed test case to perform on StakeWithUs

**Review by evilpeach — CHANGES_REQUESTED** [review]
add `.DS_Store` to .gitignore

**Comment by poomthiti** [comment]
> add `.DS_Store` to .gitignore

Done krub

**Review by evilpeach — APPROVED** [review]

**Pull request merged**